web/cypress/fixtures/config.json

```diff
@@ -1,146 +1,146 @@
 {
-    "account_recovery_mode": false,
```

Comment:
If you want to change how this file is formatted, please do it in a separate PR. It's unclear what is changed in this file.

Reply:
The pre-commit checks are failing if I do not format this file